spec/requests/api/v1/transactions_request_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe "Transactions API" do
+
+  context "GET /transactions" do
+    it "returns a list of transactions" do
+      customer = create(:customer)
+      merchant = create(:merchant)
+      invoice = create(:invoice, merchant_id: merchant.id, customer_id: customer.id)
+      transaction1 = create(:transaction, invoice_id: invoice.id)
+      transaction2 = create(:transaction, invoice_id: invoice.id)
+      transaction3 = create(:transaction, invoice_id: invoice.id)
```

Comment:
fyi you can save time by doing `transaction1, transaction2, transaction3 = create_list(:transaction, 3, invoice = invoice).